### DISCUSSION TITLE
Modifying script to install Little Endian mips on ER X

### DISCUSSION THREAD

**Discussion by picree** [jamesog/tailscale-edgeos#19] [General]
Has anyone tried modifying this script to install the mipsle version for an ER X?  Is it as simple as updating the URLs in the curl steps?

**Comment by jamesog** [comment]
If you're using the package repo it should already be doing that. If not it's probably worth opening an issue with Tailscale.

**Reply by picree** [reply]
Meaning if I already have installed tailscale_1.36.0_mipsle.tgz on an ER X then this script, if it ever runs on firstboot or post_congfig will pull that repo?  I thought this script was hard coded to pull Debian Stretch on something like an ER 4??

**Reply by jamesog** [reply]
Ah, you're probably using an old version of the installation instructions. It was updated some time ago to use Tailscale's apt package repository.

**Reply by picree** [reply]
My mistake.  Followed the instructions here and it worked just fine.  Terminal shows installation of mipsel.  Thanks for your help.
